### PR TITLE
CLSI mega Docker PR

### PR DESCRIPTION
**Pull request by henryoswald**
This is 2 layers of dockerisation rolled into 1

- The actual clsi can be run in docker, so acceptance tests pass using the new build system, and it can be used for google cloud
- Synctex now runs in docker which was needed to get the above working sharelatex/docker-runner-sharelatex#26
- Docker runner has been added into the code CLSI code base.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,79 +1,42 @@
...
-        sh 'docker run --rm -e SIBLING_CONTAINER_USER=root -e SANDBOXED_COMPILES_HOST_DIR=$(pwd)/compiles -e SANDBOXED_COMPILES_SIBLING_CONTAINERS=true -e TEXLIVE_IMAGE=$TEXLIVE_IMAGE -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd):/app sharelatex/acceptance-test-runner:clsi-6.11.2'
-        // This is a terrible hack to set the file ownership to jenkins:jenkins so we can cleanup the directory
-        sh 'docker run -v $(pwd):/app --rm busybox /bin/chown -R 111:119 /app'
-        sh 'rm -r compiles cache server.log db.sqlite config/settings.defaults.coffee'
+        sh 'DOCKER_COMPOSE_FLAGS="-f docker-compose.ci.yml" make test_acceptance'
       }
     }
-    stage('Package') {
+
+    stage('Package and publish build') {
```
Comment (henryoswald) [inline Jenkinsfile]: need to fix this

**Comment by henryoswald** [comment]
small bug with ci running unit tests I need to fix

**Comment by henryoswald** [comment]
can you try `./bin/run clsi npm install`

**Comment by briangough** [comment]
Summary of trying to debug this: mounting `/var/run/docker.sock` inside the container kills my docker daemon.  Not sure why.

**Pull request closed without merging**